src/components/header/PrimaryNav/PrimaryNav.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { NavButton } from '../NavButton/NavButton'
+import { List } from '../List/List'
+
+// assets
+import closeImg from 'uswds/src/img/close.svg'
+
+type PrimaryNavProps = {
+  items: React.ReactNode[]
+  mobileExpanded?: boolean
+}
+
+export const PrimaryNav = (
+  props: PrimaryNavProps & React.HTMLAttributes<HTMLButtonElement>
+): React.ReactElement => {
+  const { items, mobileExpanded, children, className, onClick } = props
```

Comment:
I see `onClick` is here as a native button attribute. However this component does not render a `<button>`. Because this prop is explicitly meant to toggle the mobile nav, I think the prop name should reflect that (i.e., `onToggleMobileNav`). I think the naming of the `NavButton` component should also be made clearer - it's not just a button in the nav, it's specifically used for toggling the mobile nav visibility.